.travis.yml

```diff
@@ -2,6 +2,15 @@
 sudo: required
 
 language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6"      # current default Python on Travis CI
+  - "3.7"
+  - "3.8"
+  - "3.8-dev"  # 3.8 development branch
+  - "nightly"  # nightly build
```

Comment:
@HorizonNet I have a question I want to clarify. Shall I add all of these python versions like in the link you shared to me or just some of them or a specific one?

Before these changes 3.6 version was used by default.

Reply:
Please continue to use 3.6 for now. We don't need the other one's right now.